src/sentry/static/sentry/app/views/performance/transactionSummary/content.tsx

```diff
@@ -226,6 +227,7 @@ class SummaryContent extends React.Component<Props, State> {
               end={eventView.end}
               statsPeriod={eventView.statsPeriod}
             />
+            <GlobalSdkSuggestions />
```

Comment:
If this alert is being added to the transaction summary page, should it also be added to the [transaction vitals page](https://github.com/getsentry/sentry/blob/063af51cacd748722184493723a93c910e0b868a/src/sentry/static/sentry/app/views/performance/transactionVitals/content.tsx#L183)?